### PR TITLE
feat: Remove reactive code

### PR DESCRIPTION
**Pull request by ethanmills**

### CODE REVIEW

File: src/main/kotlin/uk/gov/gdx/datashare/repository/ConsumerSubscriptionRepository.kt [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
 import java.util.*
 
 @Repository
-interface ConsumerSubscriptionRepository : CoroutineCrudRepository<ConsumerSubscription, UUID> {
+interface ConsumerSubscriptionRepository : CrudRepository<ConsumerSubscription, UUID> {
   @Query("SELECT * FROM consumer_subscription cs WHERE cs.oauth_client_id = :clientId")
-  fun findAllByClientId(clientId: String): Flow<ConsumerSubscription>
+  fun findAllByClientId(clientId: String): List<ConsumerSubscription>
 
   @Query(
```
Comment (Mjwillis) [inline src/main/kotlin/uk/gov/gdx/datashare/repository/ConsumerSubscriptionRepository.kt:13]: Could put JPA instead now

**Review by Mjwillis — APPROVED** [review]
Definitely worth discussing tomorrow, think my take is to switch. We have the option for JPA etc if we want then too and other libraries

File: build.gradle.kts [hunk 2]
```diff
@@ -39,15 +35,14 @@ dependencies {
   implementation("org.hibernate:hibernate-validator:8.0.0.Final")
 
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactive")
```
Comment (oskarwilliams378) [inline build.gradle.kts:38]: Pls look at this

Comment (ethanmills) [reply build.gradle.kts:38]: This is correct

**Pull request merged**